daemon/web/components/buttons/LogoutButton/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import '../index.sass';
+
+const LogoutButton = ({ style }) => (
```

Comment:
LogoutButton should always look the same, right? I think we can omit the `style` prop